modules/openapi-generator/src/main/resources/Java/libraries/resttemplate/ApiClient.mustache

```diff
@@ -834,7 +834,7 @@ public class ApiClient{{#jsr310}} extends JavaTimeFormatter{{/jsr310}} {
         private void logRequest(HttpRequest request, byte[] body) throws UnsupportedEncodingException {
             log.info("URI: " + request.getURI());
             log.info("HTTP Method: " + request.getMethod());
-            log.info("HTTP Headers: " + headersToString(request.getHeaders()));
+            log.debug("HTTP Headers: " + headersToString(request.getHeaders()));
```

Comment:
for your use case, you only want to skip one particular line in the log.

What about using customized templates (e.g. -t via CLI) instead?